test/expressions/diten/uintBinaryOverflow.good

```diff
@@ -1 +1 @@
-uintBinaryOverflow.chpl:1: error: Integer literal overflow: '0b10000000000000000000000000000000000000000000000000000000000000000' is too big for type 'uint64'
+uintBinaryOverflow.chpl:1: syntax error: error building numeric literal: integer literal overflow: '0b10000000000000000000000000000000000000000000000000000000000000000' is too big for a 64-bit unsigned integer
```

Comment:
Error occurs twice here; I think we'd better avoid that if we can.

Reply:
I'll just remove the "error building numeric literal: " portion as reduntant